chcemvediet/apps/inforequests/admin.py

```diff
@@ -397,3 +401,11 @@ def render_delete_form(self, request, context):
                 may cause logical errors in the inforequest history.
                 """)))
         return super(ActionAdmin, self).render_delete_form(request, context)
+
+    def delete_model(self, request, obj):
+        if request.POST:
+            if request.POST.get(u'snooze'):
```

Comment:
By sa zislo pridat podmienky aj ba BE, ze je to `EXPIRATION` alebo `APPEAL_EXPIRATION`. A tiez by sa zislo pridaj aj podmienku, ze `obj.previous_action` nie je nedefinovana. Pre istotu, ak by z nejakeho divneho dovodu sa nieco v DB domotalo, nech BE necrashne.

Reply:
:ok: